src/main/webapp/i18n/de/exam.json

```diff
@@ -652,7 +652,7 @@
             },
             "editWorkingTime": {
                 "title": "Prüfungszeit bearbeiten",
-                "label": "Änderung der Prüfungszeit",
+                "label": "Änderung der Prüfungszeit (+)",
```

Comment:
```suggestion
                "label": "Verlängern der Prüfungszeit",
```

Reply:
This was actually some kind of request to add a `+` but don't know how strong the feelings are there